src/data.json

```diff
@@ -0,0 +1,19 @@
+[
+  {
+    order:{
```

Comment:
This is compliant with Airbnb's JavaScript Guide https://github.com/airbnb/javascript#3.8